src/utils/Utils.ts

```diff
@@ -371,6 +371,15 @@ export class Utils {
     return dashCased.replace(/-([a-z])/g, g => g[1].toUpperCase());
   }
 
+  static parenthesize(str: string) {
```

Comment:
Hmm, I feel this function may be slightly over-engineered. At the moment it is behaving like a `trimAndParenthesize`.

I see only one place in this PR where we call the function, so does it make sense to put in `Utils.ts`?
Also, is handling the empty string case needed at this point in time? i.e. can `this.facetValue.formattedCount` equal `''`?

If not, I prefer to keep the logic as simple as possible for now, either in-lining, or moving the function inside `MLFacetValueRenderer` where the implementation is just line 377.